synapse/push/pusherpool.py

```diff
@@ -222,6 +223,14 @@ def _start_pusher(self, pusherdict):
         """
         try:
             p = self.pusher_factory.create_pusher(pusherdict)
+        except PusherConfigException as e:
+            logger.warning(
+                "Pusher incorrectly configured user=%s, appid=%s, pushkey=%s: %s",
+                pusherdict.get('user_name'),
+                pusherdict.get('app_id'),
+                pusherdict.get('pushkey'),
+                e,
```

Comment:
I think you generally need the type of the exception as well as the stringification

(mostly because KeyErrors are otherwise extremely confusing:

```
>>> try:
...     {}['a']
... except Exception as e:
...     print(e)
...
'a'
```

)

Reply:
Err, we know the type since we're only catching the one PusherConfigException exception?